cli/command/image/pull.go

```diff
@@ -28,7 +28,7 @@ func NewPullCommand(dockerCli command.Cli) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "pull [OPTIONS] NAME[:TAG|@DIGEST]",
-		Short: "Pull an image or a repository from a registry",
+		Short: "Download an image from an image registry into the daemon's image store",
```

Comment:
These ones (pull/push) were a bit tricky;

- trying to keep the description brief, so that it fits the terminal width and doesn't wrap in the `--help` output
- initially I had "intro the local image store", but that may be confusing, as it's "local to the daemon", which may not be "your local machine"
- at the same time, for most users, `daemon` should be mostly an implementation detail (albeit _important_ for some situations)
- "image registry" could be just "registry" (happy to hear if we think "registry" is sufficient) (we can't use "Docker Hub" here, because that just _one_ possible registry)

Reply:
How about "Download an image from a registry"? Do we need to say where it's stored? For a user, the pull command just downloads the image. I'm not sure if they need to know the details on where it's stored. I think it's more of an implementation detail?

Reply:
Yes, perhaps we should keep it simple; let me update 👍